src/main/java/com/abhyasa/java/challenge/problems/strings/Anagram.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Arrays;
+
+public class Main {
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Class name does not match filename</b></div>
<div id="fix">

The class name `Main` does not match the file name `Anagram.java`. Consider renaming the class to match the file name.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
public class Anagram {
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/100-days-of-java/pull/1#issuecomment-2535396628>#01b982</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged